data/saas/config/simon_data_config.yml

```
@@ -0,0 +1,40 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Simon Data
+  type: simon_data
+  description: A sample schema representing the Simondata connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: api.simondata.com
+    - name: api_token
```

```suggestion
    - name: api_token
      label: API token
      sensitive: True
```

You gotta give acronyms like `api` some help, otherwise they will show up as `Api` in the frontend. But most importantly, this is a credential param so it needs to be flagged as `sensitive` so it gets masked in the UI.